.github/workflows/master_default_tests.yml

```diff
@@ -0,0 +1,63 @@
+# This workflow runs anytime new commits are added to master branch.
+# It simply runs all of ObsPy's default tests on all support OS/python versions
+name: master_default_tests
+on:
+  push:
+    branches:
+      - master
+
+# Set globally scoped environmental variables.
+env:
+  OBSPY_GITHUB_BRANCH: gh-actions
```

Comment:
will this need to be changed later?

Reply:
You mean if I make a proper version branch for obspy_github_api? Yes that would be good to do :smile: